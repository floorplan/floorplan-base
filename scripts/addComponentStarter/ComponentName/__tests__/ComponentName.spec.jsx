import React from 'react';
import renderer from 'react-test-renderer';
import ComponentName from '../../ComponentName';

describe('ComponentName', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<ComponentName>Children</ComponentName>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
