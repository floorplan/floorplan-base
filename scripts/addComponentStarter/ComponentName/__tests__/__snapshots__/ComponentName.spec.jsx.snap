// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentName renders correctly 1`] = `
<div
  className="css-1bvn0qd"
>
  ComponentName
  <br />
  Children
</div>
`;
